R/uploadPackage.R

```diff
@@ -225,7 +288,11 @@ uploadPackageServer <- function(id) {
               FROM package
               WHERE name = '{uploaded_packages$package[i]}'")))
             
-            uploaded_packages$status[i] <- ifelse(found == 0, 'new', 'duplicate')
+            # if pkg was removed then reset found to 1 and keep status as "removed"
+            if(uploaded_packages$status[i] == "removed") found <- 1
+            uploaded_packages$status[i] <- ifelse(found == 0, 'new', 
+                                           ifelse(uploaded_packages$status[i] == "removed",
+                                                  'removed', 'duplicate'))
```

Comment:
Not a big deal, but since we are no longer pushing the "removed" packages through this observer, we can use the more simplistic line `uploaded_packages$status[i] <- ifelse(found == 0, 'new', 'duplicate')` from before.

Reply:
done